test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -489,6 +489,28 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
         $result.Output.Headers.'Content-Length' | Should -BeNullOrEmpty
     }
 
+    It "Invoke-WebRequest -Debug shows request information" {
```

Comment:
A debug output is not public contract. So we have no need to add tests. We will check the PR manually.
(We already have generic test that debug output works.)

Reply:
It was an easy add but I'll go ahead and remove it.